sdk/search/Azure.Search.Documents/src/Indexes/Models/SimilarityAlgorithm.cs

```diff
@@ -3,10 +3,10 @@
 
 using Azure.Core;
 
-namespace Azure.Search.Documents.Models
+namespace Azure.Search.Documents.Indexes.Models
 {
-    [CodeGenModel("AnalyzerName")]
-    public partial struct LexicalAnalyzerName
+    [CodeGenModel("Similarity")]
+    public partial class SimilarityAlgorithm
```

Comment:
Looks like it munged the diff, but where did LexicalAnalyzerName go?  Did we just "rename" it back to the default name?

Reply:
The service team renamed a bunch of models in the swagger, hence a bunch of this nature of deletes. But, yes, that diff is very weird. I didn't see that in VS's implementation of git. Weird, indeed.